demos/gaze_estimation_demo/README.md

```diff
@@ -27,7 +27,7 @@ Other demo objectives are:
 ## Running
 
 Running the application with the `-h` option yields the following usage message:
-```
+```sh
```

Comment:
It seems that your PR reverts certain changes that were made before, like this one. Please make it no longer do that.

Reply:
No, I've changed gaze_estimation_demo with according @Wovchena requests.

Reply:
That `sh` was removed intentionally by https://github.com/opencv/open_model_zoo/commit/c2a3b1880296a921867d609ea32944fbac6fbe4b. Another example, this pull request is going to add `\` in the end of lines in `demos/gaze_estimation_demo/gaze_estimation_demo.hpp`.

Reply:
`sh` was only one example of changes reverted.

Reply:
OK. Where is right readme.md? I've branched from develop

Reply:
Readme is fine now. Other files aren't. `demos/gaze_estimation_demo/gaze_estimation_demo.hpp` is one of them. May be there are more, but for other files it's harder to visually  distinguish your intentional changes and reverts.